doc/manual/manual.tex

```diff
@@ -7829,21 +7829,17 @@ \subsubsection{Inner core convection}
 \label{sec:cookbooks-inner-core-convection}
 
 \textit{This section was contributed by Juliane Dannberg, and the model setup was inspired 
-by discussions with John Rudge.}
-
-This is an example of convection in the inner core of the Earth. It uses a spherical geometry 
-and is characterized by a single material with temperature dependent density that makes it
-unstably stratified as temperature increases towards the center of the core. 
-The boundary conditions combine normal stress and normal velocity, and take into account the
-rate of phase change (melting/freezing) at the inner-outer core boundary. Gravity decreases 
-linearly from the boundary to zero at the center of the inner core. 
+by discussions with John Rudge. Additional materials and comments by Mathilde Kervazo and Marine Lasbleis}
+
+This is an example of convection in the inner core of the Earth. The model is based on a spherical geometry, with a single material. Three main particularities are constitutive of this inner core dynamics modelling: it consists of a self-gravitating sphere where the gravity decreases linearly from the boundary to zero at the center of the inner core; the boundary conditions combine normal stress and normal velocity, and take into account the rate of phase change (melting/freezing) at the inner-outer core boundary; the material is temperature dependent density,  that makes the density profile unstably stratified as temperature increases towards the center of the core.
```

Comment:
the material is temperature dependent density,  that... --> the material **has a** temperature dependent density _[no comma]_ that...

Reply:
Also, I think it would make sense to clarify the part about the self-gravitation, saying that we do not actually compute self-gravitation, but that the gravity profile is linear, and as density variations in the inner core are very small, this means that it resembles the gravity profile of a self-gravitating sphere (otherwise all of the readers from the mantle convection community will be confused, and ask us if we can do self-gravitation).

Reply:
modified!